indra/sources/omnipath/__init__.py

```diff
@@ -0,0 +1,2 @@
+from .api import process_from_web
```

Comment:
We could add a brief triplequoted description above the first import of the module and approach to accessing/processing information from Omnipath that will then show up in the documentation.